agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4397,15 +4397,62 @@ bool slave_thread::handle_autoconfiguration_wsc(Socket *sd, ieee1905_1::CmduMess
         }
     }
 
-    if (cmdu_rx.getNextTlvType() != uint8_t(ieee1905_1::eTlvType::TLV_VENDOR_SPECIFIC)) {
-        LOG(ERROR) << "Not vendor specific TLV (not Intel?)";
-        return false;
-    }
+    if (cmdu_rx.getNextTlvType() == uint8_t(ieee1905_1::eTlvType::TLV_VENDOR_SPECIFIC)) {
+        LOG(INFO) << "Intel controller join response";
+        if (!parse_intel_join_response(sd, cmdu_rx)) {
+            LOG(ERROR) << "Parse join response failed";
+            return false;
+        }
+    } else {
+        LOG(INFO) << "Non-Intel controller join response";
```

Comment:
Could be helpful for future debugging to print the manufacturer string as part of this log print.

Reply:
It's already printed together with the encrypted settings.